.github/workflows/main.yml

```diff
@@ -74,4 +74,5 @@ jobs:
     - name: Publish results to coveralls
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        COVERALLS_SERVICE_JOB_ID: ${{ github.job }}
```

Comment:
@aburrell, I'm not entirely certain this is the right way to go, but this seems to fix the coveralls issue here